### DISCUSSION TITLE
How to use GoPro wired connection

### DISCUSSION THREAD

**Discussion by GbekeAdesiyun** [gopro/OpenGoPro#380] [Q&A]
I recently bought the GoPro Hero 10 camera and would really appreciate it if you could walk me through the right way to use the wired connection to take pictures as I am having some difficulty with that. For more context, the GoPro is connected to my PC and I am running the python script found on the Open GoPro Github page (https://github.com/gopro/OpenGoPro/blob/main/demos/python/sdk_wireless_camera_control/open_gopro/demos/photo.py) .

This is the screenshot of the error I am getting by running the aforementioned script:
![image](https://github.com/gopro/OpenGoPro/assets/66129702/636a0fdb-0aa9-4e25-982f-a1d44b8ae9b0)

**Comment by GbekeAdesiyun** [comment]
Hey, just following up on this

**Comment by NickPronto** [comment]
What usb cable/port are you using?

3.0? 2.0?
